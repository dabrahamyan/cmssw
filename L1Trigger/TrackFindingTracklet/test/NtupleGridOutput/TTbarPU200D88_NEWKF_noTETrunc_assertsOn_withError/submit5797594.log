000 (5797594.000.000) 08/29 12:43:57 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_7840_2a97>
...
040 (5797594.000.000) 08/29 13:27:44 Started transferring input files
	Transferring to host: <128.142.60.117:9618?addrs=128.142.60.117-9618+[2001-1458-301-22--100-74]-9618&alias=b7s05p1446.cern.ch&noUDP&sock=slot1_19_37835_ee1c_58919>
...
040 (5797594.000.000) 08/29 13:27:44 Finished transferring input files
...
001 (5797594.000.000) 08/29 13:27:45 Job executing on host: <128.142.60.117:9618?addrs=128.142.60.117-9618+[2001-1458-301-22--100-74]-9618&alias=b7s05p1446.cern.ch&noUDP&sock=startd_5848_d1ef>
...
006 (5797594.000.000) 08/29 13:27:53 Image size of job updated: 183224
	179  -  MemoryUsage of job (MB)
	183224  -  ResidentSetSize of job (KB)
...
006 (5797594.000.000) 08/29 13:32:53 Image size of job updated: 2436172
	1050  -  MemoryUsage of job (MB)
	1075056  -  ResidentSetSize of job (KB)
...
006 (5797594.000.000) 08/29 13:41:02 Image size of job updated: 7432936
	1050  -  MemoryUsage of job (MB)
	1075056  -  ResidentSetSize of job (KB)
...
040 (5797594.000.000) 08/29 13:41:02 Started transferring output files
...
040 (5797594.000.000) 08/29 13:41:02 Finished transferring output files
...
005 (5797594.000.000) 08/29 13:41:02 Job terminated.
	(1) Normal termination (return value 134)
		Usr 0 00:11:27, Sys 0 00:00:32  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:11:27, Sys 0 00:00:32  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	47519  -  Run Bytes Sent By Job
	690  -  Run Bytes Received By Job
	47519  -  Total Bytes Sent By Job
	690  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :    68           1   2964397 
	   Memory (MB)          :  1050        2000      2000 

	Job terminated of its own accord at 2023-08-29T11:41:02Z.
...
