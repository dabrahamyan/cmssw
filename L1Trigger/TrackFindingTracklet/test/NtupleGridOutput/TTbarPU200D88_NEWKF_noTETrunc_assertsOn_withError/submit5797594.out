using geometry D88 (tilted)
Tue Aug 29 13:40:51 CEST 2023
Thread 7 (Thread 0x2b1b7a156700 (LWP 184) "cmsRun"):
#0  0x00002b1b46beeb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b1b46beebcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b1b46beec6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b1b4d6cf576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b1b4d6cf639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b1b46be8ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b1b46efbb0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b1b79f55700 (LWP 183) "cmsRun"):
#0  0x00002b1b46beeb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b1b46beebcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b1b46beec6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b1b4d6cf576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b1b4d6cf639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b1b46be8ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b1b46efbb0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b1b79d54700 (LWP 182) "cmsRun"):
#0  0x00002b1b46beeb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b1b46beebcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b1b46beec6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b1b4d6cf576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b1b4d6cf639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b1b46be8ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b1b46efbb0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b1b79b53700 (LWP 181) "cmsRun"):
#0  0x00002b1b46befe9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b1b4d40c10f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b1b4d63d2d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002b1b4d63d429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b1b46be8ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b1b46efbb0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b1b79952700 (LWP 180) "cmsRun"):
#0  0x00002b1b46efc0e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b1b4d4065c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b1b4d402d4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002b1b4d40b818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002b1b46be8ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b1b46efbb0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b1b6a09d700 (LWP 173) "cmsRun"):
#0  0x00002b1b46bf01d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b1b4dd2cad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b1b4dd2d6ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b1b467c4f90 in std::execute_native_thread_routine (__p=0x2b1b55cfb060) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b1b46be8ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b1b46efbb0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b1b48f7ee40 (LWP 155) "cmsRun"):
#0  0x00002b1b46ef0ddd in poll () from /lib64/libc.so.6
#1  0x00002b1b4dd2cdcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b1b4dd2d79c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b1b4dd3024b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b1b46e33387 in raise () from /lib64/libc.so.6
#6  0x00002b1b46e34a78 in abort () from /lib64/libc.so.6
#7  0x00002b1b467907dc in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00002b1b4679b9d6 in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
#9  0x00002b1b4679aab9 in __cxa_call_terminate (ue_header=ue_header@entry=0x2b1b886149e0) at ../../../../libstdc++-v3/libsupc++/eh_call.cc:54
#10 0x00002b1b4679b3f1 in __cxxabiv1::__gxx_personality_v0 (version=<optimized out>, actions=6, exception_class=5138137972254386945, ue_header=0x2b1b886149e0, context=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_personality.cc:685
#11 0x00002b1b46bd670f in _Unwind_RaiseException_Phase2 (exc=0x2b1b886149e0, context=0x7ffc7b6a5240, frames_p=0x7ffc7b6a5148) at ../../../libgcc/unwind.inc:64
#12 0x00002b1b46bd70b6 in _Unwind_Resume (exc=0x2b1b886149e0) at ../../../libgcc/unwind.inc:241
#13 0x00002b1b450ce043 in TDirectoryFile::Write(char const*, int, int) [clone .cold] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libRIO.so
#14 0x00002b1b4515abcf in TFile::Write(char const*, int, int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libRIO.so
#15 0x00002b1b502c6236 in TFileService::~TFileService() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libCommonToolsUtilAlgos.so
#16 0x00002b1b502ad6d9 in edm::serviceregistry::ServiceWrapper<TFileService>::~ServiceWrapper() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginCommonToolsToolsUtilAlgos_plugins.so
#17 0x00002b1b443e6bae in edm::serviceregistry::ServicesManager::~ServicesManager() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreServiceRegistry.so
#18 0x00002b1b443e6cdf in edm::serviceregistry::ServicesManager::~ServicesManager() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreServiceRegistry.so
#19 0x00002b1b443e6cdf in edm::serviceregistry::ServicesManager::~ServicesManager() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreServiceRegistry.so
#20 0x00002b1b4467de77 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#21 0x00002b1b446aa227 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#22 0x0000000000409b11 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#23 0x0000000000409739 in main ()
